system/virtualxt/virtualxt.info

```diff
@@ -1,8 +1,8 @@
 PRGNAM="virtualxt"
-VERSION="0.10"
+VERSION="0.11"
 HOMEPAGE="https://phix.itch.io/virtualxt"
-DOWNLOAD="https://github.com/andreas-jonsson/virtualxt/archive/v0.10/virtualxt-0.10.zip"
-MD5SUM="880dd53bfddcc5b91fcb4a4a2303bb1f"
+DOWNLOAD="https://github.com/andreas-jonsson/virtualxt/archive/refs/tags/v0.11.zip"
```

Comment:
breaks downloading with a browser from the sbo website

$ curl -L -o /dev/null -v https://github.com/andreas-jonsson/virtualxt/archive/refs/tags/v0.11.zip 2>&1 | grep -i content-disposition
< content-disposition: attachment; filename=virtualxt-0.11.zip

Reply:
Handled in e99f366